Removed v2023.02.17-0 from release notes (#969)
replicatedhq/replicated-docs: docs/release-notes/rn-kubernetes-installer.md

# Kubernetes Installer Release Notes

## v2023.02.17-0 - Withdrawn

Released on February 17, 2023

:::important

> v2023.02.17-0 has been removed because a new requirement in version 1.6.18 of Containerd prevents high availability installations using the EKCO internal load balancer.

Reviewer: ```suggestion
v2023.02.17-0 has been removed because an incompatibility between Containerd 1.6.18 and high availability installations using the EKCO internal load balancer.
```
